### PR TITLE
Clean up criteria sections template code and styling

### LINKED ISSUE #5255: Add new criteria and remove old criteria - template & styling
### Add new criteria and remove old criteria

- [x] Add all the criteria to match the [spreadsheet](https://docs.google.com/spreadsheets/d/1cOOzLhkoICFIPTUmxQnrH10HZz1qq5eHF51qgj1p8QM/edit#gid=0)
- [x] Remove criteria that are no longer needed 
- [x] Update the styling to match the [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Like on production, if there is no help text then that section should be hidden
- [x] If the answer to the first AI questions is `No` then the other two questions should be hidden

All styling should use styles that are already on production.

Ding will be added in  - Add "privacy" ding #5308 
Criteria can be ordered - Reorder the criteria in the product template #5253

### Desktop comp

What is required to signup?
image

What data does it collect?
image

How does it use this data?
image

How can you control your data?
image

What is the company's known track record of protecting user's data?
image

Can this product be used offline?
image

Artificial Intelligence section
image

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket #5152
- [X] Link to related ticket #5234
- [x] Link to [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included - n/a
- [x] Hover state for elements are designed - use existing styles
- [x] Other states for elements are designed (e.g., error state for form) - n/a

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5255 

Code reviewer: Please review code without whitespace changes as I fixed a ton of indentation issues & mismatched HTML tag bugs.

---
 
Please also log in to CMS, play around with the fields, and see if they show up as expected on the page.

- general product: https://foundation-s-issue-5255-zxcuts.herokuapp.com/en/privacynotincluded/products/percy-cypress/
- software product: https://foundation-s-issue-5255-zxcuts.herokuapp.com/en/privacynotincluded/products/a-conference-tool/ (Note that Sketch only has comps for general products. Please verify fields show up and in the order we want for software products.)

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -78,209 +78,184 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
...
 
-            {{ product.how_can_you_control_your_data }}
+              <div class="col-12 col-md-4">
+                {% include "fragments/signup_requirement.html" with title=personal_data_collected   value=product.personal_data_collected          %}
+              </div>
+              <div class="col-12 col-md-4">
+                {% include "fragments/signup_requirement.html" with title=biometric_data_collected  value=product.biometric_data_collected          %}
+              </div>
+              <div class="col-12 col-md-4">
+                {% include "fragments/signup_requirement.html" with title=social_data_collected     value=product.social_data_collected    %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:149]: some extra spaces at the end of these.

**Review by Pomax — CHANGES_REQUESTED** [review]
Most looks good, but we can probably remove some of the superfluous whitespacing now, and some minor improvements.

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -78,209 +78,184 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
...
         </section>
-
-        <section>
-          <h3 class="pni-product-smaller-body-bold">{% trans "Does the AI use your personal data to make decisions about you?" %}</h3>
-
-          {{ product.ai_uses_personal_data | extended_yes_no }}
+        {% endif %}
+
+
+
```
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/buyersguide/templates/product_page.html:212]: i can reduce them to just one new line in between but readibility wise it wont do much 🤔

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 3]
```diff
@@ -78,209 +78,184 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
...
-            {% endif %}
 
-            {{ product.company_track_record }}
 
-            {% if product.track_record_details %}
-              {{ product.track_record_details }}
-            {% endif %}
-          </section>
+          {% trans "How does it use this data?" as how_does_it_use_data_collected %}
+          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected   help="" %}
```
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/buyersguide/templates/product_page.html:157]: oh yea forgot to update these

File: network-api/networkapi/buyersguide/templates/product_page.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/buyersguide/templates/product_page.html]: Okay I reduced them from 3 new lines to 2 lol

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 4]
```diff
@@ -78,209 +78,184 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
...
+            {% trans "How are your recordings handled?" as handles_recordings_how %}
+            {% trans "Alerts when calls are being recorded?" as recording_alert %}
+            {% trans "Does the platform say it is compliant with US medical privacy laws?" as medical_privacy_compliant %}
+
+            {% if product.handles_recordings_how %}
+              {% include "fragments/product_criterion.html"  with  label=handles_recordings_how       value=product.handles_recordings_how                 %}
+            {% endif %}
+            {% include "fragments/product_criterion.html"  with  label=recording_alert              value=product.recording_alert|extended_yes_no      help=product.recording_alert_helptext               %}
+            {% include "fragments/product_criterion.html"  with  label=medical_privacy_compliant    value=product.medical_privacy_compliant|yes_no     help=product.medical_privacy_compliant_helptext     %}
+            {% include "fragments/product_privacy.html"    with  label=privacy_policy_links         value=""                                           links=product.privacy_policy_links                  %}
```
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/buyersguide/templates/product_page.html]: done

**Review by Pomax — APPROVED** [review]
Looks good enough to merge in to me!

**Comment by kristinashu** [comment]
Not blockers so these could be done as a follow up. Also, @sabrinang can you please confirm that these are correct before Mavis does the updates?

## Styling update requests

- [ ] "What data does it collect?" needs an "?"

_Screen caps from PR_
Screen Shot 2020-10-02 at 8 51 42 AM

- [ ] Reduce vertical space between "Camera" and "Device" (and the other places that use this spacing)

_Screen caps from PR_
image

- [ ] IA help text should be regular, not italic

_Screen caps from PR_
Screen Shot 2020-10-02 at 8 53 00 AM

- [ ] Contact info should be black with underline and turn to blue/purple with underline on hover (like on prod)

_Screen caps from PR_
Screen Shot 2020-10-02 at 8 54 29 AM

**Comment by sabrinang** [comment]
@kristinashu @mmmavis  re: comment

**What data does it collect?**
+1 add ?

**Spacing** 
In the comps I had answers 12px below questions/heading because Jen wanted that separation below to make the answer more obvious when help text was present. So it was made the same for all sections including this 3-col style one but should this be reduced everywhere? (I see it does feel far apart)

**Text in AI section**
+1 This text isn’t the last sections “help text” but rather text for all three overall AI questions. That is why it is regular and a bit further then the usual “help text” (it is similar so I can see the confusion)

**Contact links**
 +1 like on prod

**Comment by mmmavis** [comment]
@kristinashu this is ready for re-review!

I am gonna file a new ticket to make sure we are displaying criteria correctly for the software products as it requires quite a bit of the code changes and I don't want it to be a blocker.

**Review by kristinashu — APPROVED** [review]
Product page looks good!

**Pull request merged**